src/usecases/updateWardVisitTotals.js

```diff
@@ -3,7 +3,7 @@ export default ({ getDb }) => async ({ wardId, date }) => {
     const db = await getDb();
     const res = await db.any(
       "SELECT id, ward_id, total FROM ward_visit_totals WHERE ward_id = $1 AND total_date = $2",
-      [wardId, date.toISOString()]
+      [wardId, date]
     );
 
     const [existingTotal] = res;
```

Comment:
Do we need to be concerned about timing here? What if there are two ward staff adding visits at the same time? 

I guess at this stage this is an unlikely scenario

Reply:
(Summarising slack convo) For now this should be fine, longer term we'd probably opt for a more accurate performance system anyway so this will do for now